src/main/java/org/commonjava/maven/plugins/projectsrc/ProjectSourcesGoal.java

```diff
@@ -186,6 +195,16 @@ public void execute()
         }
     }
 
+    static List<String> getAssemblyFormats(String formats) {
+        List<String> parsedList = asList(formats.split(","));
```

Comment:
Couldn't we split as `formats.trim().split("\\s*,\\s*")` - wouldn't that give a single pass?

Reply:
It would loose the readability (at lease for me). Can I keep it or shall I change it your way ?

Reply:
Thats fine, thanks for the PR.